README.md

```diff
@@ -115,6 +115,7 @@ Software vs DMS build version map
 
 | jwst tag | DMS build | CRDS_CONTEXT |   Date     |          Notes                           |
 | -------- | --------- | ------------ | ---------- | -----------------------------------------|
+|  0.13.8  | B7.3.1    | 0541         | 09/05/2019 | Patch release for Build 7.3.1            |
```

Comment:
Technically isn't this a patch to B7.3, hence the Notes entry should read "Patch release for B7.3"?

Reply:
'Patch for Build 7.3 released for Build 7.3.1'?

Reply:
Minor quibble, but how about "Patch for Build 7.3 released **as** Build 7.3.1"?

Reply:
Done